packages/typescript/BUILD.bazel

```diff
@@ -12,9 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# BEGIN-INTERNAL
```

Comment:
this isn't needed - COMMON_REPLACEMENTS already mocks out the bzl_library load

Reply:
Sure. I'll remove the fencing from the load statements.